mne_icalabel/features/psd.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from mne.io.pick import _picks_to_idx
+from mne.time_frequency.psd import psd_multitaper
+from mne.utils import _check_option
+from mne.viz.ica import _prepare_data_ica_properties
+
+
+def get_psd(
+    ica,
+    inst,
+    picks=None,
+    reject="auto",
+    reject_by_annotation=False,
+    fmin=0,
+    fmax=np.inf,
+    tmin=None,
+    tmax=None,
+    bandwidth=None,
+    adaptive=False,
+    low_bias=True,
+    proj=False,
+    n_jobs=1,
+    normalization="length",
+    output="power",
+    dB=True,
+):
+    """Compute the power spectral density of the ICA components.
+
+    Parameters
+    ----------
+    ica : instance of mne.preprocessing.ICA
+        The ICA solution.
+    inst : instance of Epochs or Raw
+    picks : str | list | slice | None
+        In lists, channel type strings (e.g., ['meg', 'eeg']) will pick channels
+        of those types, channel name strings (e.g., ['MEG0111', 'MEG2623'] will
+        pick the given channels. Can also be the string values “all” to pick all
+        channels, or “data” to pick data channels. None (default) will pick all
+        channels. Note that channels in info['bads'] will be included if their
+        names or indices are explicitly provided.
```

Comment:
Same comment as for the topomap, this `picks` argument does not refer to the data channels but to the ICs.